ci/build_container/build_recipes/v8.sh

```diff
@@ -53,29 +53,26 @@ cd ../..
 
 # Patch wasm-c-api.
 
-# 1. Disable DEBUG (alloc/free accounting), since it seems to be broken
-#    in optimized builds and/or when using sanitizers.
-# 2. Disable hardcoded sanitizers.
+# 1. Disable hardcoded sanitizers.
 
 cat <<\EOF | patch -p1
 --- a/Makefile
 +++ b/Makefile
-@@ -7,10 +7,10 @@ V8_VERSION = branch-heads/7.4
- V8_ARCH = x64
+@@ -8,9 +8,9 @@ V8_ARCH = x64
  V8_MODE = release
 
--WASM_FLAGS = -DDEBUG  # -DDEBUG_LOG
+ WASM_FLAGS = -DDEBUG  # -DDEBUG_LOG
 -C_FLAGS = ${WASM_FLAGS} -Wall -Werror -ggdb -O -fsanitize=address
-+WASM_FLAGS =
 +C_FLAGS = ${WASM_FLAGS} -Wall -Werror -ggdb -O
  CC_FLAGS = ${C_FLAGS}
 -LD_FLAGS = -fsanitize-memory-track-origins -fsanitize-memory-use-after-dtor
 +LD_FLAGS =
 
  C_COMP = clang
+
 EOF
 
-# 3. Force full GC when destroying VMs.
+# 2. Force full GC when destroying VMs.
 
```

Comment:
Now that DEBUG is set can you revert the change to #if 1 ?

Reply:
Good catch, thanks! Done.